src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1549,6 +1709,22 @@ because xbuild doesn't support framework reference assemblies.
 		References="@(ReferencePath)"
 		UseManagedResourceGenerator="False"
 		DesignTimeBuild="$(DesignTimeBuild)"
+		Condition="'$(_AndroidResourceDesignerFile)' != '' And '$(AndroidUseAapt2)' != 'True'"
+	/>
+	<GenerateResourceDesigner
+		ContinueOnError="$(DesignTimeBuild)"
+		NetResgenOutputFile="$(ResgenTemporaryDirectory)\$(AndroidResgenFilename)"
+		JavaResgenInputFile="$(_GeneratedPrimaryJavaResgenFile)"
+		Namespace="$(AndroidResgenNamespace)"
+		ProjectDir="$(ProjectDir)"
+		Resources="@(_AndroidResourceDest)"
+		ResourceDirectory="$(MonoAndroidResDirIntermediate)"
+		AdditionalResourceDirectories="@(LibraryResourceDirectories)"
+		IsApplication="$(AndroidApplication)"
+		References="@(ReferencePath)"
+		UseManagedResourceGenerator="True"
+		DesignTimeBuild="$(DesignTimeBuild)"
+		Condition="'$(_AndroidResourceDesignerFile)' != '' And '$(AndroidUseAapt2)' == 'True'"
```

Comment:
Please place `Condition` at the *top*.

Given that these two `<GenerateResourceDesigner/>` invocations are nigh identical, differing only with `UseManagedResourceGenerator`, could we instead just split that part out?

```xml
<PropertyGroup>
  <_UseManagedGen Condition=" '$(AndroidUseAapt2)' == 'True' ">True</_UseManagedGen>
  <_UseManagedGen Condition=" '$(_UseManagedGen)' == '' ">False</_UseManagedGen>
</PropertyGroup>
<GenerateResourceDesigner
    ...
    UseManagedResourceGenerator="$(_UseManagedGen)"
/>
```